source/_docs/nested-docroot.md

```diff
@@ -9,16 +9,15 @@ contributors:
 
 The docroot is the directory from which your site is served. On Pantheon, this defaults to the root directory of the site's codebase. Specifying `web_docroot: true` in your <a href="/docs/pantheon-yml/#site-local-configurations-pantheonyml" data-proofer-ignore>pantheon.yml</a> file or in the <a href="/docs/pantheon-yml/#custom-upstream-configurations-pantheonupstreamyml" data-proofer-ignore>pantheon.upstream.yml</a> file in your upstream allows you to serve site files from the `web` subdirectory of your site's code repository on all Pantheon environments.
 
+<div class="alert alert-danger" role="alert">
+<h4 class="info">Warning</h4>
+<p markdown="1">Changing the settings of the `web_docroot` property will prevent you from updating your site via One-click Dashboard updates. To use this feature properly, you should switch to a [Custom Upstream](/docs/custom-upstream) that has the `web_docroot` property set in the `pantheon.upstream.yml` file.</p></div>
```

Comment:
unclear what "this feature" refers to
maybe ...
"To continue to use one-click updates, switch to..."

Reply:
Thanks @ari-gold, I updated master with your copy.